src/main/resources/db/changelog/changelog20221025_2228.sql

```diff
@@ -0,0 +1,9 @@
+-- liquibase formatted sql
+
+-- changeset nika.avalishvili:1
+CREATE TABLE benefit_type (id SERIAL PRIMARY KEY,
```

Comment:
If we know beforehand which benefit types and  calculation methods exist by default we can include them in changelog and insert them here
However it is optional